src/LightStep/LightStep.csproj

```diff
@@ -56,7 +56,7 @@
   <PropertyGroup>
       <SignAssembly>true</SignAssembly>
       <DelaySign>false</DelaySign>
-      <AssemblyOriginatorKeyFile>tracerSign.snk</AssemblyOriginatorKeyFile>
+      <AssemblyOriginatorKeyFile>../../tracerSign.snk</AssemblyOriginatorKeyFile>
```

Comment:
What does this affect?

Reply:
I moved the signing key file out of the `src/LightStep` path to the root of the project, so I updated the reference to the key file path.